parts/kubernetesmastercustomscript.sh

```diff
@@ -200,8 +200,30 @@ function configNetworkPolicy() {
     fi
 }
 
+function systemctlEnableAndCheck() {
+    systemctl enable $1
+    systemctl is-enabled $1
```

Comment:
I didn't personally, but Cole mentioned he saw this on the original issue (#853), so that's why I added this check...